lib/models/hooks.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = HooksModelFactory;
+
+HooksModelFactory.$provide = 'Models.Hooks';
+HooksModelFactory.$inject = [
+    'Model',
+    'Services.Configuration',
+    'uuid'
+];
+
+function HooksModelFactory (Model, configuration, uuid) {
+
+    return Model.extend({
+        connection: configuration.get('databaseType', 'mongo'),
+        identity: 'hooks',
+        attributes: {
+            id: {
+                type: 'string',
+                uuidv4: true,
+                primaryKey: true,
+                unique: true,
+                required: true,
+                defaultsTo: function() { return uuid.v4(); }
+            },
+            name: {
+                type: 'string',
```

Comment:
Yes, the `name` is only some words used by user to distinguish each hook. Since each hook is bind with a url, but the url sometimes is not so distinguish, such as `http://192.168.290.110:8099/test/hooks` to subscribe node topic, and use `http://192.168.290.123:8119/hooks` to subscribe poller topic, these two urls is not so easy to distinguish.


How to name a hook is owned by user, user can call it whatever he/she wants, so even a duplicated name is allowed and even user can choose not input any `name`.